playbooks/roles/ocp-config/tasks/main.yaml

```diff
@@ -61,10 +58,24 @@
     line: '\1mastersSchedulable: False'
     backrefs: yes
 
-- name: Create Chrony machineconfig files for coreOS
+- name: Setup time synchronisation with Chrony
   import_tasks: chrony.yaml
   when: chronyconfig.enabled
 
+- name: Stat ../../ocp4-helpernode/machineconfig/
+  stat:
+    path: "../../ocp4-helpernode/machineconfig/"
+  register: helpernode_mc
+
+- name: Copy machineconfig files generated from helpernode
+  copy:
+    src: "../../ocp4-helpernode/machineconfig/"
+    dest: "{{ workdir }}/manifests" 
```

Comment:
Stat and copy of specific file names *-chrony-configuration.yaml will help.

Reply:
It is in the `main.yaml`, not only specific to chrony. 
If helpernode add other machineconfig files in the future, they should be located into this directory.
Today, it is only chrony, but we never know ... what's your opinion about it ?

Reply:
Answer in the previous comment. 
> If helpernode add other machineconfig files in the future, they should be located into this directory.
Today, it is only chrony, but we never know ... what's your opinion about it ?

Reply:
I am asking to be specific because even in future what ever helpernode creates should not affect this project. Think in case the machineconfig X is not needed by the installer (eg: user specifically set enabled to false) then there will be no way to control that.

Reply:
done